src/renderer/renderElement.ts

```diff
@@ -64,6 +64,10 @@ function generateElementCanvas(
   const rc = rough.canvas(canvas);
   drawElementOnCanvas(element, rc, context);
   context.translate(-CANVAS_PADDING, -CANVAS_PADDING);
+  context.scale(
+    1 / (window.devicePixelRatio * zoom),
+    1 / (window.devicePixelRatio * zoom),
+  );
```

Comment:
This isn't technically needed since we never draw on that canvas afterwards but let's make sure we reset the transform anyway so it doesn't bite us in the future.